src/components/TodoInfo/TodoInfo.jsx

```diff
@@ -1,4 +1,14 @@
-// Add the required props
-export const TodoInfo = () => (
-  <>TodoInfo markup</>
+import { UserInfo } from '../UserInfo';
+import './TodoInfo.scss';
+
+export const TodoInfo = ({ todo }) => (
+  <article
+    className={todo.completed ? 'TodoInfo TodoInfo--completed' : 'TodoInfo'}
+  >
+    <h2 className="TodoInfo__title">
+      {todo.title}
+    </h2>
+
+    {todo.user ? <UserInfo user={todo.user} /> : null}
```

Comment:
```suggestion
    {todo.user && <UserInfo user={todo.user} />}
```
better to use logical AND  in this case